interfaces/builtin/kernel_firmware_control.go

```diff
@@ -0,0 +1,54 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+ const kernelFirmwareControlSummary = `allows adding new path for firmware location`
```

Comment:
it's not technically adding a path but overriding one that may have been set as a module parameter, so it'd suggest to rephrase the summary as `allows setting optional, custom firmware search path`, and leave a link to the docs in a comment right above

Reply:
thank you, updated.